src/diffusers/models/transformers/consisid_transformer_3d.py

```diff
@@ -0,0 +1,845 @@
+# Copyright 2024 ConsisID Authors and The HuggingFace Team. All rights reserved.
```

Comment:
@yiyixuxu So this model is mostly similar to CogVideoX except for performing cross attention with the face embeddings. I see some usages of nn.Sequential's that we don't like to have in Diffusers. Should we do the conversions and have a conversion script?